.cloudbees/testing/action.yml

```diff
@@ -54,6 +59,14 @@ runs:
       run: |
         set -uo pipefail
 
+        if echo "${CHART_LOCATION}" | grep -q '^oci://'; then
+          if [ "${CLOUDBEES_REGISTRIES_CONFIG}" ]; then
+            chart_location=$(echo -n "${CHART_LOCATION}" | sed 's|^oci://||')
+            registry-config --config "${CLOUDBEES_REGISTRIES_CONFIG}" resolve "${chart_location}" resolved_registries.out
```

Comment:
Would you mind storing the `resolved_registries.out` file within the `/tmp` directory in order to not pollute the workspace? I know you're removing the file again further down the script but that's brittle and you don't even have to do that when you store it within the container's /tmp dir in the first place.

Reply:
Done.